aries-site/src/examples/components/card/CardContentBestPractice.js

```diff
@@ -0,0 +1,110 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  Grid,
+  Heading,
+  Image,
+  NameValueList,
+  NameValuePair,
+  Page,
+  PageContent,
+  PageHeader,
+  ResponsiveContext,
+} from 'grommet';
+import { activities } from './data';
+import { Card as DesignSystemCard } from '../../templates';
+
+const columns = {
+  xsmall: ['auto'],
+  small: ['auto'],
+  medium: ['auto'],
+  large: ['medium', 'flex'],
+  xlarge: ['medium', 'flex'],
+};
+
+const activitiesColumns = {
+  xsmall: ['auto'],
+  small: ['auto'],
+  medium: ['auto'],
+  large: ['auto', 'auto'],
+  xlarge: ['auto', 'auto'],
+};
+
+export const CardContentBestPractice = ({ bestPractice = true }) => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  return (
+    <Page>
+      <PageContent>
+        <PageHeader
+          title="Manage Account"
+          subtitle="Manage your HPE Common Cloud account"
+        />
+        <Grid align="start" columns={columns[breakpoint]} gap="large">
+          <LeftPanel bestPractice={bestPractice} />
+          <Grid columns={activitiesColumns[breakpoint]} gap="medium">
+            {activities.map((activity, index) => (
+              <DesignSystemCard
+                key={index}
+                icon={activity.icon}
+                title={activity.title}
+                description={activity.description}
+                actions={
+                  <Button
+                    label={activity.action.label}
+                    href={activity.action.href}
+                    secondary
+                  />
+                }
+                level={2}
+              />
+            ))}
+          </Grid>
+        </Grid>
+      </PageContent>
+    </Page>
+  );
+};
+
+CardContentBestPractice.propTypes = {
+  bestPractice: PropTypes.bool,
+};
+
+const LeftPanel = ({ bestPractice }) => {
+  const content = (
+    <Box gap="medium">
+      <Box height="xxsmall" align="start">
+        <Image src="/att.svg" alt="AT&T logo" fit="contain" />
+      </Box>
+      <Heading level={2} size="small" margin="none">
+        AT&T Corporation
```

Comment:
I think in the Marketplace context we'd have more liberty since it is essentially advertising their service on the platform. If the cards are representative of how they are shown in Marketplace I think they are okay. If anything, its extra promotion for them.

Reply:
Done for ATT card.

Reply:
HPE Legal is pretty strict on not using other companies brands unless that company has approved our use of them. We aren't able to use them in situations like this at HPE Discover in any demos, slides, or signage. To be safe, I think we should make them generic.

Reply:
@ericsoderberghp open source like Apache Spark seems okay though. Thoughts?

Reply:
maybe. Could be fun to make up something creative though, like items in a hardware store, or a bake shop, just looking to bring a smile

Reply:
We've received feedback in the past from Design System users that "non-industry related" data in examples isn't as helpful as more "real world" type things. I am fine with removing the specific brands, but think we should stick with something that feels in the ballpark.

Reply:
Concerning the use of which brand to show in our examples. Should we approach @L0ZZI  and use the same 'faux' company brand in his North Star experience?

Reply:
Good idea, I've just pulled in his "Acme Corporation" assets.